dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3TreeTablePresenter.java

```diff
@@ -696,39 +702,14 @@ private void handleDragDetectedEvent(final Event event) {
     private void startPutJob(final Ds3Client client,
             final List<Pair<String, Path>> files,
             final String bucket,
-            final String targetDir,
-            final JobInterruptionStore jobInterruptionStore) {
-
-        final ImmutableList.Builder<kotlin.Pair<String, Path>> builder = ImmutableList.builder();
+            final String targetDir) {
+        final ImmutableList.Builder<kotlin.Pair<String,Path>> builder = ImmutableList.builder();
```

Comment:
This seems like it can be replaced with a stream call on files with a collect.  Thoughts?

Reply:
Yeah, this would be more elegant as a stream, working on it.